modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/service/test/CalendarBookingLocalServiceTest.java

```diff
@@ -831,6 +833,66 @@ public void testDeleteStagingCalendarBookingDeletesLiveCalendarBooking()
 		assertCalendarBookingsCount(stagingCalendar, 0);
 	}
 
+	@Test
+	public void testEventNotificationEscaping() throws Exception {
+		ServiceContext serviceContext = createServiceContext();
+
+		User invitingUser = UserTestUtil.addUser();
+
+		Calendar calendar = CalendarTestUtil.addCalendar(
+			invitingUser, serviceContext);
+
+		Calendar invitedCalendar = CalendarTestUtil.addCalendar(
+			_user, serviceContext);
+
+		long startTime = System.currentTimeMillis() + (Time.MINUTE * 2);
+
+		long endTime = startTime + Time.HOUR;
+
+		long firstReminder = Time.MINUTE;
+
+		serviceContext.setWorkflowAction(WorkflowConstants.ACTION_PUBLISH);
+
+		String mailBody = "[$EVENT_TITLE$], [$TO_NAME$], [$EVENT_LOCATION$]";
+		String mailSubject = "[$EVENT_TITLE$], [$TO_NAME$], [$EVENT_LOCATION$]";
+
```

Comment:
Hey @seltong, where are you using the filters that the SubscriptionSender provides to escape context variables? please use them in the template that you are creating. Thx